src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -79,7 +79,6 @@ public function getListeners(string $event = null): array
     /**
      * Checks whether an event has any registered listeners.
      *
-     * @param string $event
```

Comment:
this commit is interesting and related rule could be always enabled
but of course it should not build on the previous commits, which are BC breaking changes

Reply:
Sure, just add `Symplify\CodingStandard\Fixer\Commenting\RemoveUselessDocBlockFixer`